app/helpers/landable/application_helper.rb

```diff
@@ -5,8 +5,8 @@ def method_missing(method, *args, &block)
       super
     end
 
-    def respond_to?(method)
-      return true if method =~ /_(path|url)$/ && main_app.respond_to?(method)
+    def respond_to?(method, include_all = false)
```

Comment:
Since we don't need to check the value of this argument here, you could future-proof this by using a splat to pick up any and all extra arguments. E.g.:

```
def respond_to?(method, *args)
  ... && main_app.respond_to?(method, *args)
```